src/Fraction.php

```diff
@@ -53,35 +54,23 @@ class Fraction
      * @param integer $numerator
      * @param integer $denominator
      */
-    public function __construct($numerator, $denominator = 1)
+    public function __construct(int $numerator, int $denominator = 1)
     {
-        if (!is_int($numerator)) {
```

Comment:
This checks aren't necessary anymore since `$numerator` and `$denominator` are guaranteed to be ints.